src/commands/stac-validate/hash.worker.ts

```diff
@@ -13,7 +15,7 @@ export async function hashStream(stream: Readable): Promise<string> {
     stream.on('data', (chunk) => hash.update(chunk));
     // 0x12 - ID of sha256 multi hash
     // 0x20 - 32 bytes (256 bits) of data
```

Comment:
these comments do not make any sense here and should be moved to where the prefix is defined.